### ISSUE TITLE
Filter out archived repos

### ISSUE THREAD

**Issue by rubenvanerk** [ash-jc-allen/find-a-pr#143]
There are some archived repos in `config/repos.php`, I think it would be good if they are ignored when crawling. They also should be hidden in the sidebar.

**Comment by ash-jc-allen** [comment]
Yeah I agree on this one. I'm guessing that we can probably get the "archived" status from the API when we're crawling the repos?

**Comment by rubenvanerk** [comment]
![image](https://user-images.githubusercontent.com/20305359/192756852-1f4bd899-772b-4dfa-bbd9-fdadcf48265d.png)

Looks like there is. There's also a `has_issues` that we could use to filter the repos to crawl.

**Comment by nuernbergerA** [comment]
will do a PR
